dandi/consts.py

```diff
@@ -119,6 +119,9 @@ class DandiInstance(NamedTuple):
     "dandi-api-local-docker-tests": DandiInstance(
         None, None, f"http://{instancehost}:8000/api"
     ),
+    "dandi-archive-local-docker": DandiInstance(
+        f"http://{instancehost}:8085", None, f"http://{instancehost}:8000/api"
```

Comment:
The fact that both this and dandi-api-local-docker-tests have `http://{instancehost}:8000/api` as the API URL (is that intentional?) will likely lead to issues with `known_instances_rev` and possibly some other code.  What about just setting the GUI URL for dandi-api-local-docker-tests to `http://{instancehost}:8085`?

Reply:
I thought about that, but decided not to "pretend" that we have one scenario whenever we have two. Not yet sure indeed what could be manifestations of ambiguity - will need to check later.

Reply:
ok, decided to just agree -- pushed adding web ui url to that existing instance instead

Reply:
uff -- tests liked it more before ;)
```
FAILED dandi/cli/tests/test_download.py::test_download_api_instance_in_dandiset
FAILED dandi/cli/tests/test_instances.py::test_cmd_instances - AssertionError...
FAILED dandi/tests/test_dandiarchive.py::test_parse_api_url[http://localhost:8085/dandiset/000002-parsed_url17]
```
so I would need to check detail one way or another